src/main/kotlin/lotto/domain/LottoBuyer.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class LottoBuyer(lottoNumbers: ArrayList<Lotto>) {
```

Comment:
```suggestion
class LottoBuyer(lottoNumbers: List<Lotto>) {
```
꼭 ArrayList를 받을 필요 없이 List 인터페이스의 구현체라면 모두 허용해도 될 것 같아요 😄